src/averageAges.js

```diff
@@ -56,6 +73,15 @@ function calculateWomenAverageAge(people, withChildren) {
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
   // write code here
+  let children = people.filter(person => {
+    return person.mother && people.map(m => m.name).includes(person.mother);
```

Comment:
```suggestion
    return person.mother && people.some(m => m.name === person.mother);
```
Зачем там делать map?